components/src/core/components/CardTable/Cell/types.ts

```diff
@@ -23,7 +23,7 @@ export interface RowItem {
   isDisabled?: boolean;
 }
 
-export interface ActionCellEvent extends MouseEvent {
+export interface ActionCellEvent extends UIEvent {
```

Comment:
The `ActionCellEvent` can be made generic by extending it from the `UIEvent`. This will enable using this type for both `KeyBoardEvent`s and `MouseEvent`s scenarios.